include/Addon/Upseller.php

```diff
@@ -0,0 +1,32 @@
+<?php namespace EmailLog\Addon;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Handles upsell messages.
+ */
+
+ class Upseller{
+
+   	/**
```

Comment:
Add the `Loadie` interface to this class and then add the `load` method.

After that add it to the loadie list at https://github.com/sudar/email-log/blob/dev/2.4.0/load-email-log.php#L55

Reply:
Fixed.